### PR TITLE
Write new Task: Attaching Handlers to Container Lifecycle Events.

### PR DESCRIPTION
**Pull request by steveperry-53**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2225)

### CODE REVIEW

File: docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+### Defining postStart and preStop handlers
+
+In this exercise, you create a Pod that has one Container. The Container has handlers
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md]: When does the postStart fire?  Before the containers are started?  After?  At the same time?

File: docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+for the postStart and preStop events.
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md]: Does the system wait for the preStop command to finish before stopping the containers?  How do this relate to graceful termination seconds and any shutdown signals sent to the container processes?

**Comment by steveperry-53** [comment]
@pwittrock, Addressed your comments. Still having trouble with Travis failing.

**Comment by pwittrock** [comment]
@steveperry-53 Travis is a nightmare

**Comment by pwittrock** [comment]
@steveperry-53 Thanks for addressing the comments.  Your explanation really clears things up.

**Pull request merged**